README.md

```diff
@@ -48,6 +47,6 @@ Parts of the lyrics editor in this ruleset were inspired by them.
 
 - [Appveyor](https://www.appveyor.com/), [CodeFactor](https://www.codefactor.io/) and [Github action](https://github.com/features/actions) for providing free `CI`/`CD` service.
 
-- [Figma](https://www.figma.com/), for fast creating assets like logo or icon.
+- [Figma](https://www.figma.com/), for fast creation of assets like logos or icon.
 
-- [Miro](https://miro.com/). Used to declare the complex question by drawing the graph.
+- [Miro](https://miro.com/). Used for project management and planning.
```

Comment:
Miro is a tool for drawing something like flow chart, not for project management.

Reply:
And it used to help me decide the architecture for some features.

Reply:
I updated that miro line just a few seconds ago